test/replcompletions.jl

```diff
@@ -744,6 +744,13 @@ function test_dict_completion(dict_name)
     c, r = test_complete(s)
     @test c == Any[":α]"]
 end
+
+if Base.have_color # The test is only meaningful when Base.have_color==true
```

Comment:
it doesn't add a noticeable amount of time to test or anything does it? and if it's not exactly redundant with something already tested elsewhere, don't think it's harmful to run regardless